frame/ethereum/src/lib.rs

```diff
@@ -229,6 +229,15 @@ pub mod pallet {
 
 			0
 		}
+
+		fn on_runtime_upgrade() -> Weight {
+			frame_support::storage::unhashed::put::<EthereumStorageSchema>(
+				&PALLET_ETHEREUM_SCHEMA,
+				&EthereumStorageSchema::V2,
+			);
+
+			0
```

Comment:
Should be `db_weight.write`

Reply:
Let me get that in a separate PR since we actually lack full weight handling in the pallet.